### PR TITLE
RTC: add modal for desktop notifications

### PR DESCRIPTION
**Pull request by paglias**

### CODE REVIEW

File: common/locales/en/generic.json [hunk 1]
```diff
@@ -184,8 +184,8 @@
     "wonChallengeShare": "I won a challenge in Habitica!",
     "achievementShare": "I earned a new achievement in Habitica!",
     "orderBy": "Order By <%= item %>",
-
-    "you": "you",
```
Comment (crookedneighbor) [inline common/locales/en/generic.json:188]: Is this used anywhere else? Or did _you_ add it for the original notifications pr?

**Comment by crookedneighbor** [comment]
Other than the promise vs callback thing, this looks good to me!

File: common/locales/en/generic.json (hunk 1 above)
Comment (paglias) [reply common/locales/en/generic.json:188]: I added it in the original PR and was suggested by translators to change it

**Comment by crookedneighbor** [comment]
Looks great

**Pull request merged**